src/sass/partials/_hero.scss

```diff
@@ -56,17 +59,147 @@
 .hero__lady_with_icecream {
   display: none;
 }
+.hero__arrow {
+  display: none;
+}
+//tabet
+@media (min-width: 768px) {
+  .hero-container {
+    background-color: $IBgColor;
+
+    background-image: url(../images/ellipse.png), url(../images/ellipse.png);
+    background-repeat: no-repeat, no-repeat;
+    background-position: 730px 9px, 730px 75px;
+    background-size: 16px 16px, 16px 16px;
+    position: relative;
+  }
+  .hero-container::before {
+    background-image: url(../images/ice-cream_tablet.png);
+    position: absolute;
+    background-repeat: no-repeat;
+    height: 336px;
+    width: 100vw;
+    top: -23px;
+    left: 378px;
+    content: '';
+    z-index: 3;
+  }
+  .hero-container:after {
+    background-image: url(../images/background-circle_tablet.png);
+    position: absolute;
+    background-size: 324px 322px;
+    background-repeat: no-repeat;
+    height: 100%;
+    width: 100vw;
+    top: -42px;
+    left: 307px;
+    content: '';
+    z-index: 2;
+  }
+  .hero__slogan-buttons {
+    padding-top: 15px;
+    padding-bottom: 66px;
+    padding-left: 14px;
+  }
+  .hero__slogan {
+    text-transform: uppercase;
+    height: 80px;
+    width: 168px;
+    font-family: $fontSecondary;
+    font-size: 22px;
```

Comment:
Slogan's font-size isn't correct, according to the layout it's different for each version

Reply:
@KamilaDudzik but according to the layout on Figma the font size should be 22px